index.html

```diff
@@ -145,29 +148,151 @@ <h2 class="content-home-heading" id="content-home-heading">The best destinations
 
 </section>
 <!--Sign up-->
-<section>
-<aside class="profile-bg">
-  <div class="profile" >
-  <button class="profile-user"><img src="img/user.png" alt=""></button>
-  <div class="profile-p-b">
-    <h3>Sign up and get 20% off your first booking</h3>
-    <button class="profile-sing-button">Sign up</button>
-  </div>
-  <button class="profile-clouse"><img src="img/Vector.svg" alt=""></button>
-  </div>
-</aside>
+<section class="sign-up-order">
+  <aside class="profile-bg">
+    <div class="profile" >
+      <button class="profile-user"><img src="img/user.png" alt=""></button>
+      <div class="profile-p-b">
+        <h3>Sign up and get 20% off your first booking</h3>
+        <button class="profile-sing-button">Sign up</button>
+      </div>
+      <button class="profile-clouse"><img src="img/Vector.svg" alt=""></button>
+    </div>
+  </aside>
 
   <aside class="mob-profile-bg">
     <div class="mob-profile" >
-    <button class="mob-profile-user"><img class="mob-profile-user-i" src="img/Vector.png" alt=""></button>
-    <img class="tripHouseMob-img" src="img/tripHouseMob.svg" alt="tripHouseMob">
-    <div class="mob-profile-h-a">
-      <h3 class="mob-profile-h" >Sign up in the app and get 20% off your first booking</h3>
-      <a class="mob-profile-a">Install</a>
-    </div>
+      <button class="mob-profile-user"><img class="mob-profile-user-i" src="img/Vector.png" alt=""></button>
+      <img class="tripHouseMob-img" src="img/tripHouseMob.svg" alt="tripHouseMob">
+      <div class="mob-profile-h-a">
+        <h3 class="mob-profile-h" >Sign up in the app and get 20% off your first booking</h3>
+        <a class="mob-profile-a">Install</a>
+      </div>
     </div>
   </aside>
 </section>
+
+<!-- Guests reviews -->
+<section>
+  <div class="guests-bg">
+    <h2 class="content-home-heading-des">Guests reviews</h2>
+    <h2 class=" content-home-heading-mob">Reviews</h2>
+    <div class="reviews">
+      <div class="reviews-width" id="reviews-width">
+        <div class="content-reviews" id="content-reviews">
+          <div class="content-reviews-img">
+            <figure class="hotel">
+              <img src="img/hotel.png" alt="hotel">
+              <figcaption> <p>From 250 USD</p> </figcaption>
+            </figure>
+            <div class="content-reviwes-img-fig">
+              <div class="location">
+                <div class="location-p">
+                  <p>Ubud Bali Resort&SPA</p>
+                  <p>Bali, Indonesia</p>
+                </div>
+                <div class="estimation">
+                  <img src="img/score.svg" alt="">
```

Comment:
Please, use text and styles instead of svg for a score! On real projects you will be take this info from a server and it will never be an image